cloud-auto-registration.yaml

```diff
@@ -15,14 +15,14 @@ data:
     targetIdentities:
     - apiauthorizationpolicy
     longDescription: |-
-      Define an API authorization policy to allow the auto-registration of the enforcer using an AWS cloud identity.
+      Define an API authorization policy to allow the auto-registration of the enforcer using an {{ .Values.cloud_provider }} cloud identity.
```

Comment:
I am not sure you can have it in the description. Did you try it?

Reply:
Nope. So should I remove it from the longDescription? For sure description needs to be modified and it can be updated. I will leave longDescription as it is. Thanks

Reply:
Yes, if it can't work let just have a generic description

Reply:
OK I imported the recipe into sandbox and tried and did not observe any issues.

Reply:
For safe side I will leave long description as it is

Reply:
No, that's not accurate to keep it as is, because it says it is creating an AWS cloud registration
Just remove "AWS" :)